### PR TITLE
[Guide] Created OmniMIDI Guide and added contrib.rocks to the main page

### LINKED ISSUE #16: [Guide] OmniMIDI Setup
**What is your guide?**
OmniMIDI getting started guide

**Who will this help?**
This will help a complete beginner set up OmniMIDI and provide information on a variety of things such as the WinMM Patch

**What type of guide is this?**
This will be a step-by-step guide with screenshots on how to install, setup, and use OmniMIDI

**Additional Context**
There will also need to be a Recommended Soundfonts page, which I will set up in a future issue.

### PR DESCRIPTION
**Pull request by snowydqys**
Completed the getting started guide, configuration of soundfonts guides, and moved the WinMM Patch from PFA section to OmniMIDI section.

Also added [contrib.rocks](https://contrib.rocks/preview?repo=OvertureMIDI%2FGuides) to the main page of the docs

### CODE REVIEW

**Comment by snowydqys** [comment]
Also I left an empty recommended soundfonts page as I'm not sure what soundfonts would be best for beginners. So I leave that to someone else

File: docs/OmniMIDI/Soundfonts/configuration.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Adding Soundfonts
+
+Adding soundfonts is pretty simple in OmniMIDI. Typically, soundfonts come in 2 different formats: `sfz` and `sf2`. `sfz` format soundfonts come with a "samples" folder and must be included with the `sfz`. `sf2` is one file only and does not need this extra folder. Both accomplish the same thing but in different ways.
+
+!!! info "Help! I have no soundfonts!"
+
+    If you have no soundfonts available, you are free to Google soundfonts and use those. Although not recommended as some soundfonts do not pair well with Black MIDIs. Instead, you can use our [list of recommended soundfonts](#recommended-soundfonts) to find yourself a good soundfont to use! Other resources will include Black MIDI Community Discord servers.
```
Comment (6gh) [inline docs/OmniMIDI/Soundfonts/configuration.md:7]: There is no title with #recommended-soundfonts on this page

File: docs/index.md [hunk 2]
```diff
@@ -18,3 +18,13 @@ As I don't have _a lot_ of experience in blacking, some guides might not be the
 You can help contribute by submitting a pull request on the [Github repo for this project](https://github.com/6gh/Black-MIDI-Guides).
 
 You are free to help out in any possible way you think you can expand, or improve on this project. Things such as creating new guides, elaborating on existing guides, or even just fixing any simple typos or grammatical errors are very welcome! So thank you if you choose to contribute to the project!
+
+### Contributors
```
Comment (6gh) [inline docs/index.md:22]: This is neat!

**Review by 6gh — APPROVED** [review]
I see a tiny bit of mistakes but otherwise good guide! I will clean up the mistakes myself in the `cleanup` branch

**Comment by 6gh** [comment]
I will also handle the recommended sound fonts. Thank you for contributing!

**Pull request merged**